base.rb

```diff
@@ -1,11 +1,14 @@
-def do_something
+def do_somethin
   
 end
 
-def do_something2(a, b)
+def do_something2(a, b, c)
   
 end
 
-def do_something3(a: b)
+def do_something3(c: b)
   
 end
+
+do_something
```

Comment:
__[development][method_definition_validator]__
* do_something is undefined. Did you mean? do_somethin